configWebMaintained.js

```diff
@@ -43,7 +43,29 @@ const quickSurveyScenarios = utils.makeScenariosForSkins(
 	[ 'vector', 'vector-2022', 'minerva' ]
 );
 
+const nearbyScenarios = utils.makeScenariosForSkins(
+	[
+		{
+			label: 'Nearby the WMF office',
+			path: '/wiki/Special:Nearby',
+			hash: '#/coord/37.789167,-122.403333',
+			hashtags: [ '#nearby' ],
+			selectors: [ '#mw-content-text' ]
+		},
+		{
+			label: 'Nearby error state',
+			path: '/wiki/Special:Nearby',
+			hash: '#/coord/0,-700',
+			hashtags: [ '#nearby' ],
+			selectors: [ '.mw-content-container' ]
```

Comment:
I see a number of "Element not found for capturing: .mw-content-container" errors when running pixel locally for the "web-maintained" group which I believe is caused by `vector` and `minerva` skins not having the `mw-content-container` element

<img width="621" alt="Screen Shot 2022-09-28 at 3 24 20 PM" src="https://user-images.githubusercontent.com/3496279/192891499-c89a1edd-7e77-4d7b-90ed-652d7027a69d.png">

<img width="1718" alt="Screen Shot 2022-09-28 at 3 20 07 PM" src="https://user-images.githubusercontent.com/3496279/192891234-e31df0e4-9891-4f28-b171-84e3bbca195c.png">

Reply:
Whoops. Unstaged changes !

Reply:
I'm still getting element not found for "error state" scenarios although its bizarre because I can see the element in the HTML, can you see those as well or is it only my setup?

Reply:
Blah its because of typo lol (see below)